test/multiverse/suites/grpc/Envfile

```diff
@@ -1,10 +1,13 @@
+# encoding: utf-8
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
 # frozen_string_literal: true
 
 suite_condition('grpc gem requires CRuby 2.5+') do
   RUBY_VERSION >= '2.5.0' && RUBY_PLATFORM != 'java'
 end
 
-instrumentation_methods(:chain, :prepend)
+instrumentation_methods(:prepend)
```

Comment:
Let's make sure this gets reverted before merge!

Reply:
reverted with a389f1de67f742ca97d941906cd5dbe5fc2627ae